airflow/utils/db_cleanup.py

```diff
@@ -370,3 +382,19 @@ def run_cleanup(
                 session=session,
             )
             session.commit()
+
+
+@provide_session
+def export_archived_records(export_format, output_path, session: Session = NEW_SESSION):
```

Comment:
Hmm .. Should we instead have a flag on `db clean` command to `export` before clean/archive? Or do we expect users to run this command separately to db clean too?

Reply:
That was the first thing we did but it looks like a separate command is better...pushing all the contents of the archived table to filesystem. See https://github.com/apache/airflow/pull/29058#discussion_r1087901858

Reply:
Cool, I will leave that upto you